src/utils/WithToggleControl.jsx

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+/**
+ * Provides boolean isChecked prop to wrapped component.
+ * Prop name can also be customized if need be
+ *
+ * @param {React.element} ToggleComponent - the component to wrap
+ * @returns {React.element}
+ */
+export default function withToggleControl(WrappedComponent, propName = 'isChecked') {
+	/**
+	 * @module WithToggle
+	 */
+	class WithToggle extends React.Component {
+		constructor(props) {
+			super(props);
+
+			this.state = { isActive: props['isActive'] };
```

Comment:
I'm sort of surprised that `Prettier` or ESLint didn't catch this, but style preference is `props.isActive`